2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'meme_experiments', 'entity': 'vietnguyen', 'root_dir': '/content/SSLMemes', 'run_id': '2f5dns2j', 'sweep_param_path': '/content/SSLMemes/wandb/sweep-68f4kv9d/config-2f5dns2j.yaml', 'sweep_id': '68f4kv9d'}
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'src/train.py', 'program': '/content/SSLMemes/src/train.py'}
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_init.py:_log_setup():405] Logging user logs to /content/SSLMemes/wandb/run-20221011_033050-2f5dns2j/logs/debug.log
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_init.py:_log_setup():406] Logging internal logs to /content/SSLMemes/wandb/run-20221011_033050-2f5dns2j/logs/debug-internal.log
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_init.py:init():439] calling init triggers
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_init.py:init():443] wandb.init called with sweep_config: {'batch_size': 144, 'dropout': 0, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.03902489013162763, 'optimizer': 'sgd'}
config: {}
2022-10-11 03:30:50,230 INFO    Thread-7  :17105 [wandb_init.py:init():492] starting backend
2022-10-11 03:30:50,231 INFO    Thread-7  :17105 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-11 03:30:50,235 INFO    Thread-7  :17105 [backend.py:ensure_launched():219] starting backend process...
2022-10-11 03:30:50,291 INFO    Thread-7  :17105 [backend.py:ensure_launched():225] started backend process with pid: 17313
2022-10-11 03:30:50,292 INFO    Thread-7  :17105 [wandb_init.py:init():501] backend started and connected
2022-10-11 03:30:50,301 INFO    Thread-7  :17105 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 144, 'dropout': 0, 'epochs': 1, 'hidden_size': 256, 'learning_rate': 0.03902489013162763, 'optimizer': 'sgd'}
2022-10-11 03:30:50,309 INFO    Thread-7  :17105 [wandb_init.py:init():565] updated telemetry
2022-10-11 03:30:50,430 INFO    Thread-7  :17105 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-10-11 03:30:51,410 INFO    Thread-7  :17105 [wandb_run.py:_on_init():1759] communicating current version
2022-10-11 03:30:51,474 INFO    Thread-7  :17105 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-11 03:30:51,474 INFO    Thread-7  :17105 [wandb_init.py:init():625] starting run threads in backend
2022-10-11 03:30:54,974 ERROR   Thread-7  :17105 [wandb_init.py:init():981] error
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_init.py", line 954, in init
    run = wi.init()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_init.py", line 636, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/interface/interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/interface/interface_shared.py", line 432, in _communicate_run_start
    result = self._communicate(rec)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/interface/interface_shared.py", line 222, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/usr/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
Exception
